src/pages/settings/Security/TwoFactorAuth/Steps/CodesStep.js

```diff
@@ -0,0 +1,146 @@
+                >
+                    <View style={styles.mv3}>
+                        <Text>{translate('twoFactorAuth.codesLoseAccess')}</Text>
+                    </View>
+                    <View style={styles.twoFactorAuthCodesBox(dimensionsProps)}>
+                        {account.isLoading ? (
+                            <View style={styles.twoFactorLoadingContainer}>
+                                <ActivityIndicator color={themeColors.spinner}/>
+                            </View>
+                        ) : (
+                            <>
+                                <View style={styles.twoFactorAuthCodesContainer}>
+                                    {Boolean(account.recoveryCodes) &&
+                                        _.map(account.recoveryCodes.split(', '), (code) => (
+                                            <Text
+                                                style={styles.twoFactorAuthCode}
+                                                key={code}
+                                            >
+                                                {code}
+                                            </Text>
+                                        ))}
+                                </View>
+                                <View style={styles.twoFactorAuthCodesButtonsContainer}>
+                                    <PressableWithDelayToggle
+                                        text={translate('twoFactorAuth.copy')}
+                                        textChecked={translate('common.copied')}
+                                        icon={Expensicons.Copy}
+                                        inline={false}
+                                        onPress={() => {
+                                            Clipboard.setString(account.recoveryCodes);
+                                            setIsNextButtonDisabled(false);
+                                        }}
+                                        styles={[styles.button, styles.buttonMedium, styles.twoFactorAuthCodesButton]}
+                                        textStyles={[styles.buttonMediumText]}
+                                    />
+                                    <PressableWithDelayToggle
+                                        text={translate('common.download')}
+                                        icon={Expensicons.Download}
+                                        onPress={() => {
+                                            localFileDownload('two-factor-auth-codes', account.recoveryCodes);
+                                            setIsNextButtonDisabled(false);
+                                        }}
+                                        inline={false}
+                                        styles={[styles.button, styles.buttonMedium, styles.twoFactorAuthCodesButton]}
+                                        textStyles={[styles.buttonMediumText]}
```

Comment:
```suggestion
                                        textStyles={styles.buttonMediumText}
```

Reply:
Failed prop type error in the console, it expects an array